### PR TITLE
Add a page listing the shortcuts

### PR DESCRIPTION
**Pull request by alix-tz**
There are probably more in the version after v. 0.13.4b

### CODE REVIEW

**Comment by alix-tz** [comment]
would fix #49

File: docs/shortcuts.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+- ++control+backspace++: reset the zoom
+- ++right-button+"(hold and drag)"++: move on the zoomed image
+- ++escape++ while drawing a line: cancel the drawing and stop drawing
+- ++control+z++ after creating, deleting or modifying a line: undo the creation/deletion/edition
+- ++control+y++: undo the creation/deletion/edition
+- ++control+a++: select all the lines/regions
+- ++shift+left-button+"(hold and drag)"++: draw a selection over lines, regions or points
+- ++shift+left-button++ (on a line or a region): add or remove a line from the active selection
+- ++control+left-button+"(hold and drag)"++ on an active selection: move all the selection together
+- ++delete++ with an active selection: delete the selected lines or the regions
```
Comment (FloChiff) [inline docs/shortcuts.md:64]: Still no for Mac

File: docs/shortcuts.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+- ++control+delete++ with an active selection of points: delete the selected points, but not the whole line or the whole region
```
Comment (FloChiff) [inline docs/shortcuts.md:65]: That doesn't work either

Comment (FloChiff) [reply docs/shortcuts.md:65]: Update: fn + ctrl + backspace work to delete selected points

File: docs/shortcuts.md (hunk 1 above)
Comment (FloChiff) [reply docs/shortcuts.md:64]: Update: fn + backspace work to delete selected lines or regions

File: docs/shortcuts.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+### Edition Page - Source Image panel
+
+- ++control+backspace++: reset the zoom
+- ++right-button+"(hold and drag)"++: move on the zoomed image
+
+If image annotation is available:
+
+- ++escape++: cancel the annotation in progress or cancel the modification
+- ++delete++ with an active annotation: delete the annotation
```
[the remark this answers is not in the document]
Comment (FloChiff) [reply docs/shortcuts.md:49]: Update: fn + backspace work to delete the annotation

File: docs/shortcuts.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+Typical URL: `base_url/document/{doc_id}/images/`.
+
+- hold ++shift++ after clicking on a thumbnail checkbox and before clicking on another one: select an interval of document parts
+
+## Edition Page
+
+Typical URL: `base_url/document/{doc_id}/part/{doc_part}/edit/`.
+
+- ++control+arrow-right++ OR ++page-down++: go to the next element (or document-part)
```
[the remark this answers is not in the document]
Comment (FloChiff) [reply docs/shortcuts.md:33]: Update: fn + page-up or -down work to go to the previous or next element

[the remark this answers is not in the document]
Comment (alix-tz) [reply docs/shortcuts.md:33]: I'm not entirely sure which shortcut your refer to.

Would the following be correct?

- ++shift+control+arrow-right++ OR ++fn+page-down++: go to the next element (or document-part)
- ++shit+control+arrow-left++ OR ++fn+page-up++: go to the previous element
- ++shift+control+1++: toggle on/off the Metadata panel
- ++shift+control+2++: toggle on/off the Source image panel
- ++shift+control+3++: toggle on/off the Segmentation panel
- ++shift+control+4++: toggle on/off the Transcription panel
- ++shift+control+5++: toggle on/off the Text panel


image

[the remark this answers is not in the document]
Comment (FloChiff) [reply docs/shortcuts.md:33]: Everything is correct, but for all the shortcut referring to panel (toggle on/off), the shift is not necessary for mac, ctrl+number is enough

**Comment by alix-tz** [comment]
sorry for this ugly commit tree! It's ready for merge I think, and we can always update later anyway if necessary

**Pull request merged**